### DISCUSSION TITLE
Graph only showing one instance of a relationship

### DISCUSSION THREAD

**Discussion by AttaboySawyer** [neo4j-contrib/neovis.js#363] [Q&A]
Hello,

I am following example configs with my own data, and am running into a problem of the graph only ever showing a single relationship:
![image](https://github.com/neo4j-contrib/neovis.js/assets/83662258/6c3dde25-830e-458a-b46c-72f02d2cf533)
Even if there are multiple types of relationships, it only ever shows one. 

I have been following example configs and others examples, and am wondering if there is a requirement in the relationship config or the labels config that may be causing this error. Here is an example of my relationship config:
```js
relationships: {
          "HAS": {
            value: "idNumber",
            [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
              function: {
                label: rel => rel.type,
                title: () => "HAS"
              }
            }
          }
```
Have others come across this issue? I'm not sure if this is an issue with the config or my data.

**Comment by thebestnom** [comment]
Most likely it's cypher prblem, can you send the initialCypher or check if there are errors in the console?

**Comment by AttaboySawyer** [comment]
```js
initialCypher: "MATCH p=(m)-[r1]->(n)-[r2]->(i) RETURN m,r1,n,r2,i LIMIT 50"
```
And console:
![image](https://github.com/neo4j-contrib/neovis.js/assets/83662258/05c011f9-6d77-4a5a-9425-724391145a23)

**Comment by thebestnom** [comment]
Umm WTH 😅 can you enable debug log? 😅

**Comment by thebestnom** [comment]
Maybe the LIMIT is wrong? 😅
https://neo4j.com/developer/kb/limiting-match-results-per-row/

**Comment by AttaboySawyer** [comment]
Seems to be an issue with my data. Started from scratch creating data purely with Cypher queries and now it is showing relationships:
![image](https://github.com/neo4j-contrib/neovis.js/assets/83662258/66bc9c7a-62c4-497d-9bbb-a4fb1467b640)
Before, I had created some data using the Neo4j explorer, so maybe somewhere down the line it messed with node IDs or something🤷‍♂️ Don't know for sure but thanks for your help!

**Reply by thebestnom** [reply]
Are you sure it wasn't the LIMIT, cause it doesn't make sense 😅

**Reply by AttaboySawyer** [reply]
Yea even when I removed the limit it returned the same as before
